Formula/quantlib.rb

```diff
@@ -38,12 +37,20 @@ class Quantlib < Formula
   def install
     ENV.cxx11 if build.cxx11?
     if build.head?
-      Dir.chdir "QuantLib"
       system "./autogen.sh"
     end
+    if MacOS.version >= :mavericks
+      ENV.append "LDFLAGS", "-stdlib=libstdc++ -mmacosx-version-min=10.6"
+      ENV.append "CXXFLAGS", "-stdlib=libstdc++ -mmacosx-version-min=10.6"
```

Comment:
I don't know
quantlib website has:
> Users have reported linking problems since Mac OS X 10.9; the solution (thanks to Albert Azout for pointing it out) seems to be to set the environment flags CXXFLAGS and LDFLAGS to -stdlib=libstdc++ -mmacosx-version-min=10.6 before compiling.